.github/workflows/e2e-tests.yml

```
@@ -242,6 +235,13 @@ jobs:
     needs: [gate, authorize]
     if: needs.gate.outputs.proceed == 'true'
     runs-on: ubuntu-latest
+    # Singleton against the one physical VPS runner. cancel-in-progress: false — a maintainer
+    # preempts via the Actions tab, never a comment. queue: max FIFO-queues runs instead of
+    # the default `single`, which cancels the waiting run when a newer one queues.
+    concurrency:
+      group: e2e-tests-singleton
+      cancel-in-progress: false
+      queue: max
```

actionlint (1.7.12)

[error] unexpected key "queue" for "concurrency" section. expected one of "cancel-in-progress", "group"

(syntax-check)